composer.lock

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Nejsem si jistý, jestli chceš přidávat soubory `composer.json` a `composer.lock`. 

Do budoucna s tím nemám problém, ale tak jak jsi to přidal teď to asi nemá význam, ne?